src/main/java/net/masterthought/cucumber/ReportBuilder.java

```diff
@@ -264,4 +194,111 @@ private void generateErrorPage(Exception exception) throws IOException {
         errorPage.generatePage();
         System.out.println(exception);
             }
+
+    public static final class Builder {
```

Comment:
I would make this class outer and no static -  that would be helpful to maintain it